contract/src/actions.rs

```diff
@@ -339,7 +335,6 @@ impl Contract {
 
         for asset_amount in in_assets {
             let asset = self.internal_unwrap_asset(&asset_amount.token_id);
-            account.add_affected_farm(FarmId::Supplied(asset_amount.token_id.clone()));
```

Comment:
No need to add this manually, since the supplied farm added in `internal_set_asset`